doc/usecases/template.md

```diff
@@ -2,10 +2,11 @@
 
 ## Summary
 
-- **Scope:** <Scope, e.g. Authentication>
-- **Level:** <Level, e.g. User Goal>
-- **Actors:** <Actors, e.g. Anonymous User>
+- **Scope:** <e.g. Authentication>
+- **Level:** <Cloud | Kite | Sea | Underwater | Clam>
```

Comment:
It is from Alistair Cockburn in "Writing effective use cases". For me it is not a strict requirement that it must be from these 5 and its definitely not the idea that anyone starts writing the same use case 5 times on all levels :wink: 

But before everyone introduces their own arbitrary names for levels, it is better to stick to those. I think they are easily understandable without reading the whole book.

Reply:
> I think they are easily understandable without reading the whole book.

Well, **I** don't really understand them... I mean, yes it's obvious that "Cloud" is very high-level and "Clam" is super low-level. But what exactly that would be in relation to Elektra is pretty unclear. IMO we need at least one example for Cloud and Clam each.

IMO there should also be a delineation at the low-level since we probably don't want all the details in use cases and at some point there should be a transition to decisions.